bin/concurrency/plot.gpi

```diff
@@ -9,7 +9,7 @@ set multiplot layout 1,3 title sprintf("test: %s\n%s\n%s", test_name, test_descr
 set datafile separator ','
 
 set xrange[0:test_time*1000]
-set xtics rotate by -45 1000
+set xtics rotate by -45 10000
```

Comment:
I kinda like the tics very second though for long time intervals I can see that it would make the tics harder to read. I wonder if there is a way we could scale the tics but ensure they sit on full seconds (rather than partial).

Reply:
And I find the numbers squashed together at the bottom very hard to distinguish. Is it markers in the graph you prefer having every second, or the labels on the edge? I think we can set them separately.

Reply:
Yeah, that makes sense. I'd be content with having the markers be every second the the labels every 10. That should match the common scales at which we run these tests. I like the second markers as rate limits are usually oriented around seconds so it makes it easier to see that.